docs/dx_tables.md

```diff
@@ -1,549 +1,74 @@
-|          |      |             |     |                          |-------------|
-| NPHL     |    0 |           0 |   2 | CODE TABLE               |-------------|
-| NPSM     |    0 |           0 |   2 | CODE TABLE               |-------------|
-| NPQC     |    0 |           0 |   4 | FLAG TABLE               |-------------|
-|          |      |             |     |                          |-------------|
-| QCEVR    |    0 |           0 |   4 | CODE TABLE               |-------------|
-| QMGP     |    0 |           0 |   4 | CODE TABLE               |-------------|
-| QMAT     |    0 |           0 |   4 | CODE TABLE               |-------------|
-| QMDD     |    0 |           0 |   4 | CODE TABLE               |-------------|
-| QMWN     |    0 |           0 |   4 | CODE TABLE               |-------------|
-| QMCA     |    0 |           0 |   1 | CODE TABLE               |-------------|
-| UARDC    |    0 |           0 |   8 | CODE TABLE               |-------------|
-| QMPR     |    0 |           0 |   4 | CODE TABLE               |-------------|
-| CORN     |    0 |           0 |   3 | CODE TABLE               |-------------|
-| QMST     |    0 |           0 |   4 | CODE TABLE               |-------------|
-|          |      |             |     |                          |-------------|
-| BULTIM   |    0 |           0 |  64 | CCITT IA5                |-------------|
-| BBB      |    0 |           0 |  64 | CCITT IA5                |-------------|
-| RRSTG    |    0 |           0 |  64 | CCITT IA5                |-------------|
-|          |      |             |     |                          |-------------|
-`------------------------------------------------------------------------------'
-</pre>
-
-<br>
-
-As noted during the discussion of subroutine openbf(), every BUFR file that is presented to the
-BUFRLIB software, either for input (reading/decoding) or output (writing/encoding) purposes, must
+Every BUFR file must
 have DX BUFR tables associated with it, unless the 'SEC3' decoding option is specified during the
 call to openbf(). For all other cases, DX table information must be pre-defined and made available
-to the software via call argument LUNDX during the call to openbf(). In the case of an existing
-BUFR file, the DX tables information may be embedded within the first few BUFR messages of the
+to the software via call argument LUNDX during the call to openbf(). The DX tables information may be embedded within the first few BUFR messages of the
 file itself. Otherwise, a separate ASCII text file containing the necessary DX tables information
-must be supplied, such as the example shown above. It is extremely important that any such file
-not only be syntactically correct but also complete, in the sense that all necessary mnemonics
+must be supplied, such as the example shown above. Such files
+must be syntactically correct and also complete, in the sense that all necessary mnemonics
 must exist and be fully-defined.
```

Comment:
change to "example shown below", since you've now moved the example to the end of the file ;-)

Reply:
Done.